src/components/menu/components/dishes/components/categoryDishes/components/dish/Dish.js

```diff
@@ -0,0 +1,61 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import dishReader from './readers/dishReader';
+import _noop from 'lodash/noop';
+
+//Components
+import DishImage from '../../../../../../../../commonComponents/dishImage';
+import DishButtton from './components/dishButtton';
+
+//CSS
+import './dish.css';
+
+function Dish(props) {
+  const { dish, quantities, handleEvents } = props;
+  const dishId = dishReader.id(dish);
+  const name = dishReader.name(dish);
+  const price = dishReader.price(dish);
+  const isVeg = dishReader.isVeg(dish);
+  const description = dishReader.description(dish);
+
+  return (
+    <div className='dish'>
+      <div>
+        <DishImage isVeg={isVeg} />
+        <h3 className='styledH3'>{name}</h3>
+        <p className='styledP'>₹ {price}</p>
+        <div className='styledDiv'>{description}</div>
+      </div>
+      <div>
+        <DishButtton
```

Comment:
you don't need this as a component, you're just using it a single time and inside there is only if and else condition rendering